item-spec/examples/sample-full.json

```diff
@@ -16,6 +16,18 @@
   },
   "properties": {
     "datetime": "2016-05-03T13:22:30.040Z",
+    "title": "A CS3 item",
+    "license": "proprietary",
```

Comment:
If license is proprietary there should be a link to the license...

Reply:
Good call. I'll change it to something known then.